packages/nimble-components/src/wafer-map/tests/wafer-map-matrix.stories.ts

```diff
@@ -115,6 +117,21 @@ const componentWaferWithDieOrientation = ([
 >
 </nimble-wafer-map>`;
 
+const componentWaferResize = (
+    size: number
+): ViewTemplate => html`<nimble-wafer-map
+        class="size${size}"

```

Comment:
Instead of adding a `<style>` block that targets each wafer map with a unique class, another approach would be to add inline style on each element, something like this:
```
html`<nimble-wafer-map
        style="width: ${size}px; height: ${size}px"
```

I think that would be a little bit cleaner: fewer `<style>` blocks on the page and less code to implement.

Reply:
I did consider this, however I am not a big fan of inline styling ... still, I applied it since indeed in this case it would reduce the footprint.

Reply:
Yeah, inline styling is a bad idea for production code since it's harder to override, reason about, format, read, etc. But for small snippets of test code I think it's ok.